cmake/modules/ClangFlags.cmake

```diff
@@ -82,8 +82,8 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
       -Wno-gnu-anonymous-struct
       -Wno-nested-anon-types
     )
-  target_compile_options(tvm_objs PRIVATE $<$<COMPILE_LANGUAGE:CXX>: ${warning_opts}>)
-  target_compile_options(tvm_runtime_objs PRIVATE $<$<COMPILE_LANGUAGE:CXX>: ${warning_opts}>)
+  target_compile_options(tvm PRIVATE $<$<COMPILE_LANGUAGE:CXX>: ${warning_opts}>)
+  target_compile_options(tvm_runtime PRIVATE $<$<COMPILE_LANGUAGE:CXX>: ${warning_opts}>)
```

Comment:
This causes the warning options to only be used when linking not when compiling. I don't think that is the behavior we want.